### ISSUE TITLE
Request: Disallow extra spaces directly inside parens

### ISSUE THREAD

**Issue by scriby** [palantir/tslint#235]
Our coding standards state that ifs should be formatted like so:

``` ts
if (x === y) {
  //do some stuff
}
```

I would like to disallow extra spaces inside the parens for consistency:

``` ts
if ( x === y ) {
  //do some stuff
}
```

Similarly with other control structures (I would be ok if the rule were to never allow a space directly insoie parens).

**Comment by ajafff** [comment]
Fixed by #2959
